spec/fixtures/multiline-soa

```diff
@@ -0,0 +1,7 @@
+osuosl.org      IN SOA  ns1.auth.osuosl.org. hostmaster.osuosl.org. (
```

Comment:
Can we have 2 more versions of this ?

* One where there is a comment character after the opening parentheses and followed by comment.
* One with hyphen in the ns server name.

I believe I and @jldugger discussed this. If this has been already eliminiated as a good testcase, please ignore.